db/seeds/evaluatieonderzoek/questionnaires/evaluatieonderzoek.rb

```diff
@@ -27,7 +33,7 @@
                        type: :dropdown,
                        label: 'RMC-regio',
                        title: 'Bij welke RMC-regio hoort uw school?',
-                       tooltip: '<p>Als u het niet zeker weet, kunt u op dit kaartje kijken:</p><img width="90%" src="/evaluatieonderzoek/rmcregios.png">',
+                       tooltip: '<p>Als u het niet zeker weet, kunt u op dit kaartje kijken:</p><img height="640" src="/evaluatieonderzoek/rmcregios.png">',
```

Comment:
ik weet niet precies hoe dat plaatje eruit ziet, maar willen we dit niet ook op mobile supporten? en `px` mist

Reply:
Ja dit heb ik nog op mijn lijst staan. Het is nu vet brak hoe het werkt. Ik maak er een ticket van.

Reply:
px moet trouwens niet volgens https://www.w3schools.com/TAGS/att_img_width.asp